src/orion/core/cli/plot.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Module running the plot command
+========================
+
+Exposes the interface for plotting for command-line usage.
+
+"""
+import argparse
+import logging
+
+import orion.core.io.experiment_builder as experiment_builder
+from orion.core.cli import base as cli
+from orion.storage.base import get_storage
+import orion.plotting.base
+
+# c'est quoi que ça fait ça?
+#from orion.storage.base import get_storage
+
+
+log = logging.getLogger(__name__)
+DESCRIPTION = "Produce plots for Oríon experiments"
+
+
+
+def add_subparser(parser):
+    """Add the subparser that needs to be used for this command"""
+    plot_parser = parser.add_parser("plot", help=DESCRIPTION, description=DESCRIPTION)
+
+    cli.get_basic_args_group(plot_parser)
+
+    plot_parser.add_argument(
+        "kind",
+        type=str,
+        help="kind of plot to generate. "
+        " Pick one among ['lpi', 'partial_dependencies', 'parallel_coordinates', 'regret']"
+    )
+
+    plot_parser.add_argument(
+        "-t",
+        "--type",
+        type=str,
+        default="png",
+        help="type of plot to return. "
+        " Pick one among ['png', 'jpg', 'jpeg', 'webp', 'svg', 'pdf', 'html', 'json']"
+        " (default: png)",
+    )
+
+    plot_parser.add_argument(
+        "-o",
+        "--output",
+        type=str,
+        default="",
+        help="path where plot is saved. "
+        " Will override the `type` argument."
+        " (default is {exp.name}_{kind}.{type})"
+    )
+
+    plot_parser.add_argument(
+        "--scale",
+        type=float,
+        default=None,
+        help="more pixels, but same proportions of the plot. "
+        " Reference is 1.0."
+        " Overrides value of 'scale' in plotly.io.write_image."
+    )
+
+    plot_parser.set_defaults(func=main)
+
+    return plot_parser
+
+
+def main(args):
+    """Starts an application that will generate a plot."""
+    config = experiment_builder.get_cmd_config(args)
+
+    # Note : If you specify no argument at all, the default behavior
+    #        is to plot the lpi as <experiment_name>.png .
+
+
+    # What does this do?    
+    #experiment_builder.setup_storage(config.get("storage"))
+    
+    experiment = experiment_builder.build_from_args(args)
+
+    assert args['kind'] in ['lpi', 'partial_dependencies', 'parallel_coordinates', 'regret']
```

Comment:
You could drop this when passing choices to argparse

Reply:
Good point. I didn't know that was an option. I'll do it for both 'type' and 'kind'.